Signal/src/AppDelegate.m

```diff
@@ -1130,17 +1130,9 @@ - (void)checkIfAppIsReady
 
     [AppVersion.sharedInstance mainAppLaunchDidComplete];
 
-    [Environment.shared.contactsManager setup];
```

Comment:
These deletions are key changes - each of these components has taken over responsibility for "queuing up" their own setup methods.